README.md

```diff
@@ -161,6 +161,7 @@ Returns a promise with the string ID of the upload.  Will reject if there is a c
 |`enabled`|boolean|Optional|Enable or diasable notifications|`{ enabled: true }`|
 |`autoClear`|boolean|Optional|Autoclear notification on complete|`{ autoclear: true }`|
 |`notificationChannel`|string|Optional|Sets android notificaion channel|`{ notificationChannel: "My-Upload-Service" }`|
+|`enableRingTone`|boolean|Optional|Sets whether or not to enable the notification sound when the upload gets completed with success or error|`{ enableRingTone: true }`|
```

Comment:
Please indicate this is android-only.  Also if you could alphabetize the props, I'd appreciate it

Reply:
@StevePotter I've added this to the header. The complete notification object is android only